plugins/modules/pnp_intent.py

```diff
@@ -318,14 +323,16 @@ def get_site_details(self):
                 params={"name": self.want.get("site_name")},
             )
         except Exception:
+            self.log("Site {0} not found".format(self.want.get("site_name")), "CRITICAL")
             self.module.fail_json(msg="Site not found", response=[])
 
         if response:
-            self.log(str(response))
+            self.log("Site details of {0} received are {1}".format(self.want.get("site_name"), str(response)), "DEBUG")
             site = response.get("response")
             if len(site) == 1:
                 site_id = site[0].get("id")
                 site_exists = True
+                self.log("Site Id is {0} and site name is {1}".format(site_id, self.want.get("site_name")), "INFO")
```

Comment:
Can we say 
                self.log("Site Name: {1},  Site ID: {0}".format(site_id, self.want.get("site_name")), "INFO")

Reply:
Resolved